src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRulesForm/dataPrivacyRulesFormSelectControl.tsx

```diff
@@ -9,20 +9,37 @@ type Props = Pick<
   'value' | 'placeholder' | 'name' | 'onChange' | 'options' | 'isDisabled'
 >;
 
-const DataPrivacyRulesPanelFormSelectControl = ({...props}: Props) => (
-  <SelectControl
-    {...props}
-    isSearchable={false}
-    styles={{
-      control: (provided: {[x: string]: string | number | boolean}) => ({
-        ...provided,
-        minHeight: '40px',
-        height: '40px',
-      }),
-    }}
-    openOnFocus
-    required
-  />
-);
+class DataPrivacyRulesPanelFormSelectControl extends React.Component<Props> {
+  componentDidMount() {
+    if (!this.selectRef.current) {
+      return;
+    }
+
+    if (this.selectRef.current?.select?.inputRef) {
+      this.selectRef.current.select.inputRef.autocomplete = 'off';
+    }
+  }
+
+  selectRef = React.createRef<typeof SelectControl>();
+
+  render() {
+    return (
+      <SelectControl
+        {...this.props}
+        isSearchable={false}
+        styles={{
+          control: (provided: {[x: string]: string | number | boolean}) => ({
+            ...provided,
+            minHeight: '40px',
+            height: '40px',
+          }),
+        }}
+        ref={this.selectRef}
```

Comment:
Should we add an `autocomplete` prop to the selectcontrol so this can be set more directly? I looked into whether or not this is something this is something that react-select makes easy. It doesn't appear to be trivial as we would to override the `Input` component with custom props

```javascript
import { components } from 'react-select';

const Input = props => <components.Input {...props} autoComplete="off" />;

<SelectControl
    { ... }
    components={{
        Input
    }}
/>
```

Reply:
I wished I could simply pass the autocomplete prop to the SelectControl component (react-select),  but since I am not using the isSearchble prop, in final I the rendered component is the DummyInput which doesn't accept other props. I would have to create a PR in their repository...

I also tried to fix this issue in the component SelectControl, but since I many other parts of our app were broken, I decided to modify the SelectControl only inside of dataPrivacyRulesFormSelectControl.tsx